omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/PatientIdentifierResource1_8.java

```diff
@@ -248,7 +248,7 @@ public void purge(PatientIdentifier delegate, RequestContext context) throws Res
 	 */
 	@Override
 	public NeedsPaging<PatientIdentifier> doGetAll(Patient parent, RequestContext context) throws ResponseException {
-		return new NeedsPaging<PatientIdentifier>(parent.getActiveIdentifiers(), context);
+		return new NeedsPaging<PatientIdentifier>(parent.getActiveIdentifiers(), context.getIncludeAll());
```

Comment:
Instead of patient.getActiveIdentifiers(), use patient.getIdentifiers()
if context.getIncludeAll() is true, then just return this list.
Else build and return a new list that does not have voided ones.
Does this make sense?